pyspeckit/spectrum/models/ammonia.py

```diff
@@ -179,41 +181,43 @@ def ammonia(xarr, tkin=20, tex=None, ntot=14, width=1, xoff_v=0.0,
                                        (Crot-Brot)*Jpara**2)/(kb*tkin))
         Zortho = 2*(2*Jortho+1)*np.exp(-h*(Brot*Jortho*(Jortho+1)+
                                            (Crot-Brot)*Jortho**2)/(kb*tkin))
+        Qpara = Zpara.sum()
+        Qortho = Zortho.sum()
         for linename in line_names:
             if ortho_dict[linename]:
                 orthoparafrac = fortho
```

Comment:
Notationally, this should probably just become `orthofrac` since `orthoparafrac` invokes the idea of ortho-to-para ratio (i.e., OPR).  Implementation looks fine.